### PR TITLE
Add tagging functionality to Jenkinsfile

### PR DESCRIPTION
**Pull request by velychkovskyi**

### CODE REVIEW

**Review by vecinomio — APPROVED** [review]
Looks Good To Me

File: Jenkinsfile [hunk 1]
```diff
@@ -11,12 +11,13 @@ pipeline {
     timestamps()
   }
   parameters {
-    string(defaultValue: '0.0.0', description: 'A version of Release', name: 'VERSION')
     booleanParam(name: 'BuildApp', defaultValue: true, description: '')
     booleanParam(name: 'Test', defaultValue: true, description: '')
```
Comment (mike-unitskyi) [inline Jenkinsfile]: wouldn't extract this as separate parameter (should be the same as `build`)

File: Jenkinsfile [hunk 2]
```diff
@@ -11,12 +11,13 @@ pipeline {
...
     booleanParam(name: 'Delivery', defaultValue: false, description: '')
     booleanParam(name: 'Tagging', defaultValue: false, description: '')
```
Comment (mike-unitskyi) [inline Jenkinsfile]: would combine `Tagging` and `Delivery` to `release`.

File: Jenkinsfile [hunk 3]
```diff
@@ -11,12 +11,13 @@ pipeline {
     timestamps()
   }
   parameters {
-    string(defaultValue: '0.0.0', description: 'A version of Release', name: 'VERSION')
     booleanParam(name: 'BuildApp', defaultValue: true, description: '')
```
Comment (mike-unitskyi) [inline Jenkinsfile]: would rename to `build`

**Review by mike-unitskyi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**